psoc/grobot_workspace/grobot_module_controller.cydsn/dht11.h

```diff
@@ -0,0 +1,23 @@
+/* ========================================
+ *
+ * Copyright YOUR COMPANY, THE YEAR
+ * All Rights Reserved
+ * UNPUBLISHED, LICENSED SOFTWARE.
+ *
+ * CONFIDENTIAL AND PROPRIETARY INFORMATION
+ * WHICH IS THE PROPERTY OF your company.
+ *
+ * ========================================
+*/
+
+
+#ifndef dht11_h
+#define dht11_h
+    
+#define dhtok 0
```

Comment:
Avoid using ```#defines``` when possible. I would make these constants and declare them in the .c file.